### ISSUE TITLE
Translations are not accepted

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-admin#2336]
Several users reported translations are not honored in the WooCommerce Admin pages, while they appear correctly in the menu.

Thread in the .org repo:

https://wordpress.org/support/topic/translation-is-not-accepted/

**Comment by justinshreve** [comment]
I was able to reproduce this on an older version of WordPress (unfortunately I clicked upgrade before noting which version) -- but things seem to be loading correctly for me with `5.2.1` with the latest translations pulled down.

<img width="1482" alt="Screen Shot 2019-06-05 at 6 14 51 PM" src="https://user-images.githubusercontent.com/689165/58972884-e1013700-87be-11e9-9de2-5271a23390ab.png">

From some debugging, it looks like `$relative` was returning `false` for me before (https://core.trac.wordpress.org/browser/trunk/src/wp-includes/l10n.php?rev=45254#L981), which prevented the .json files from loading. On 5.2.1, this is no longer occurring. From trac, it seems like there have been fixes around this in the last few months.

I've asked for some version information on the .org thread.

**Comment by timmyc** [comment]
From the feedback in the forum thread, this still appears to be a problem for a few users, so I'm going to reopen this and drop it back in the sprint backlog.

**Comment by rrennick** [comment]
Another thread since 0.14.0 was released https://wordpress.org/support/topic/translation-not-working-77/#post-11679623

I wonder if poEdit might be missing the php file in the languages folder.

**Comment by rrennick** [comment]
I translated a string  in my dev install that was used in both PHP & script. Confirming that translations in PHP are working but not working with script.

**Comment by jeffstieler** [comment]
I don't believe there is any bug here, just some missing documentation.

Just to double check that, follow these steps to verify:

1. Set language to Español
1. Go to Dashboard > Updates (Escritorio > Actualizaciones)
1. Scroll down to Traducciones
1. Click button to "Actualizar traducciones" if necessary
1. Verify that all strings in WooCommerce Admin are in Spanish

**Comment by rrennick** [comment]
It's still not working for me. I translated `Orders` as `Canadians`:

<img width="504" alt="Screen Shot 2019-07-08 at 1 44 57 PM" src="https://user-images.githubusercontent.com/343847/60827606-f6370000-a186-11e9-90cd-2b5f0754aff4.png">

**Comment by rrennick** [comment]
> I don't believe there is any bug here, just some missing documentation.

Agreed. Okay to close.

**Comment by apmwebdev** [comment]
Not sure if this is the same issue, but I'm seeing translations not working now. 
![image](https://user-images.githubusercontent.com/30607075/86847546-1cc99880-c072-11ea-87f9-79dad06aef31.png)

That is Brazilian Portuguese on my local test site, and a user is experiencing the same issue in [this forum thread](https://wordpress.org/support/topic/cant-translate-woocommerce-analytics-dashboard/). I saw the same strings not being translated thing when I tried German as well. The translations are up to date, and the LT says WC is 100% translated for both languages. I'm using the 2020 theme.

**Comment by apmwebdev** [comment]
Another thread about this [here](https://wordpress.org/support/topic/missing-translations-18/).

**Comment by lucas-filippi** [comment]
I'm experiencing the same issue

**Comment by fuchsfelipel** [comment]
I'm also experiencing the same issue

**Comment by davidpragmatica** [comment]
I'm also experiencing the same issue

**Comment by alexcrd15** [comment]
Hi, I'm experiencing the same issue with Italian translations

**Comment by Frisoni** [comment]
Just to check with others, did this issue works for someone?
I have 2 differente store and both of them lost all translations after this update.
Even the few ones that used to work.

**Comment by jeffstieler** [comment]
Hello @Frisoni,

This issue should have been resolved with version `1.4.0` of WooCommerce Admin, which is included in WooCommerce `4.4`.

Are you experiencing the same problem on the latest version?

**Comment by Frisoni** [comment]
Hi @jeffstieler ,

I was watching this issue since before version 4.4.
I had the same problem as @apmwebdev screenshot ( https://user-images.githubusercontent.com/30607075/86847546-1cc99880-c072-11ea-87f9-79dad06aef31.png ). Note that Inbox and some translations were correct, but most translations in JS files wasn't.

Since 4.4 update, all string are in english. Check my screenshot (now in 4.5)
![image](https://user-images.githubusercontent.com/12817135/92931472-e1ed3380-f419-11ea-8834-da632a6e573c.png)

I did create a fresh install of WooCommerce in 4.4 and it was all in english.

Really don't know whats going on.

**Comment by jeffstieler** [comment]
@Frisoni - there may be an issue with the translations loading for the WooCommerce Admin package included in WooCommerce core. If you aren't using the separate WooCommerce Admin plugin, can you try installing that and updating translations?

Thanks!

**Comment by Frisoni** [comment]
@jeffstieler every translation is update. No translation button is shown.
I just created a new install only of WP, then installed just WooCommerce and translations are in english.
You can login to check, if you wish: https://wordpress-176158-1479291.cloudwaysapps.com/wp-admin/?wtlwp_token=1a86bb54253c03266111764f2dde7b51

**Comment by Frisoni** [comment]
@jeffstieler Did some tests here:
When WooCommerce is installed, translation files from woocommerce-admin are NOT downloaded to language/plugins folder.
I installed WooCommerce Admin plugin, and all translation files were downloaded and admin pages translated to portuguese.
Deleted Woo Admin plugin -> everything in english again.
Reupload translation files do language/plugins folder Without Woo Admin plugin and it is still in english.

Maybe this is more of a woocommerce issue.
I found this one that seems to be the problem: https://github.com/woocommerce/woocommerce/issues/26994
